source/reference/minio-mc/mc-support-logs-enable.rst

```diff
@@ -0,0 +1,61 @@
+==========================
+``mc support logs enable``
+==========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support logs enable
+
+Description
+-----------
+
+Use the :mc-cmd:`mc support logs enable` command to allow real-time MinIO logs to upload to |subnet|.
+
+.. include:: /includes/common-mc-support.rst
+   :start-after: start-support-logs-opt-in
+   :end-before: end-support-logs-opt-in
+   
+.. include:: /includes/common-mc-support.rst
+   :start-after: start-minio-only
+   :end-before: end-minio-only
+
+.. note:: 
+
+   By default, MinIO does not scrub the logs uploaded to SUBNET.
+
+   To hide sensitive information in the logs, start the server with the :mc-cmd:`~mc server --anonymous` flag.
```

Comment:
I wonder if we should note something like

"MinIO employs an aggressive scrubbing algorithm which may produce logs with reduced visibility into the deployment. MinIO Engineering may later request unredacted logs if required for ongoing support cases."

Reply:
👍